internal/infrastructure/kubernetes/proxy/resource.go

```diff
@@ -204,13 +229,18 @@ func expectedContainerVolumeMounts(deploymentSpec *egcfgv1a1.KubernetesDeploymen
 			Name:      "sds",
 			MountPath: "/sds",
 		},
+		{
+			Name:      "coredump",
+			MountPath: "/tmp/",
```

Comment:
instead of `/tmp` , should we use `/cores` ?

Reply:
Considering whether to mount a general directory to meet other requirements later, not just core files. So in the above design, /tmp was mounted by default.

Reply:
its due to `/tmp` that we need to add a security context right ? im trying to remove this constraint

Reply:
Modifying the sysctl kernel parameters requires permission, maybe the securityContext permission can be set smaller, let me check it.